.github/workflows/publish-release.yml

```diff
@@ -24,7 +24,10 @@ jobs:
                 env:
                     packageUser: ${{ github.actor }}
                     packagePAT: ${{ secrets.GITHUB_TOKEN }}
-                run: ./gradlew build -x check -x test
+                run: |
+                    git config --global user.name ${{ secrets.BALLERINA_BOT_USERNAME }}
+                    git config --global user.email ${{ secrets.BALLERINA_BOT_EMAIL }}
+                    ./gradlew build -x check -x test
```

Comment:
Shall we disable examples build also in this step?

Reply:
+1

Reply:
Fixed with 99b153a2d88ba3c7cc21e108e40f6d0b8b0f63f6